R/class-network.R

```diff
@@ -0,0 +1,48 @@
+# Network class
+check_network <- function(object) {
+
+  errors <- character()
+
+  # Check that all nodes in links are in nodes
+  nodesInLinks <- NodeList(union(getSourceNodes(object@links), getTargetNodes(object@links))) # may become a method later if i find i use it elsewhere
```

Comment:
i have to think about the list of node lists. I can see how it'd be nice for some of the organization but my first reaction is that it will be harder to work with and no truer to the abstract concept than having properties.

Reply:
I think Id like to hear more about longitudinal and multilayer, for a sense where we might go, before forming an opinion on the partition slot vs list of nodeLists..